lib/scala/docs-generator/src/main/scala/org/enso/docs/generator/DocParserWrapper.scala

```diff
@@ -0,0 +1,63 @@
+package org.enso.docs.generator
+
+import java.io.File
+import org.enso.syntax.text.{DocParser, Parser}
+import org.enso.syntax.text.docparser._
+
+/** Defines useful wrappers for Doc Parser.
+  */
+object DocParserWrapper {
+
+  /** Generates HTML of docs from Enso program.
+    */
+  def run(program: String): String = {
+    val parser   = new Parser()
+    val module   = parser.run(program)
+    val dropMeta = parser.dropMacroMeta(module)
+    val doc      = DocParserRunner.createDocs(dropMeta)
+    val code     = DocParserHTMLGenerator.generateHTMLForEveryDocumented(doc)
+    code
+  }
+
+  /** Generates HTML from Documentation string.
+    */
+  def runOnPureDoc(comment: String): String = {
+    val doc  = DocParser.runMatched(comment)
+    val html = DocParserHTMLGenerator.generateHTMLPureDoc(doc)
+    html
+  }
+
+  /** Called if file doesn't contain docstrings, to let user know that they
+    * won't find anything at this page, and that it is not a bug.
+    */
+  def mapIfEmpty(doc: String): String = {
+    var tmp = doc
```

Comment:
I would avoid this `var` and just create new immutable variables for each result there.